frameworks/helloworld/integration/tests/test_utils.py

```diff
@@ -68,10 +71,13 @@ def success_predicate(response):
     return spin(fn, success_predicate)
 
 
-def install(additional_options = {}):
+def install(package_version=None, package_name=PACKAGE_NAME, additional_options = {}):
```

Comment:
Looks like the default value of the argument is being used (`PACKAGE_NAME`), and not any other value that could be passed in via that parameter.